src/test/VoteMonitor.Api.IntegrationTests/VoteMonitor.Api.IntegrationTests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AssemblyName>VoteMonitor.Api.IntegrationTests</AssemblyName>
+    <PackageId>VoteMonitor.Api.IntegrationTests</PackageId>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <UserSecretsId>TBR</UserSecretsId>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <ProjectGuid>{0e63cb9a-486a-48f7-91b2-b1e2766088d3}</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.6" />
+    <PackageReference Include="Moq" Version="4.13.1" />
```

Comment:
Did we ever consider other mocking frameworks? For example, NSubstitute which is less verbose comparing with Moq.

Reply:
@czlatea I will agree here with you but let's see if moq will be a problem